apps/etterlatte-saksbehandling-ui/client/src/components/tilbakekreving/utbetalinger/TilbakekrevingVurderingPerioderSkjema.tsx

```diff
@@ -74,129 +77,162 @@ export function TilbakekrevingVurderingPerioderSkjema({
-                      {Object.values(TilbakekrevingSkyld).map((skyld) => (
-                        <option key={skyld} value={skyld}>
-                          {teksterTilbakekrevingSkyld[skyld]}
-                        </option>
-                      ))}
-                    </Select>
-                  </Table.DataCell>
-                  <Table.DataCell key="resultat">
-                    <Select {...register(`values.${index}.ytelse.resultat`)} label="Resultat" hideLabel={true}>
-                      <option value="">Velg..</option>
-                      {Object.values(TilbakekrevingResultat).map((resultat) => (
-                        <option key={resultat} value={resultat}>
-                          {teksterTilbakekrevingResultat[resultat]}
-                        </option>
-                      ))}
-                    </Select>
-                  </Table.DataCell>
-                  <Table.DataCell key="tilbakekrevingsprosent">
-                    <TextField
-                      {...register(`values.${index}.ytelse.tilbakekrevingsprosent`, {
-                        pattern: { value: /^[0-9]+$/, message: 'Kun tall' },
-                        min: { value: 0, message: 'Må være større enn 0' },
-                        max: { value: 100, message: 'Kan ikke være større enn 100' },
-                      })}
-                      label=""
-                      error={errors.values && errors.values[index]?.ytelse?.tilbakekrevingsprosent?.message}
-                      hideLabel={true}
-                    />
-                  </Table.DataCell>
-                  <Table.DataCell key="rentetillegg">
-                    <TextField
-                      {...register(`values.${index}.ytelse.rentetillegg`, {
-                        pattern: { value: /^[0-9]+$/, message: 'Kun tall' },
-                        maxLength: { value: 10, message: 'Beløp kan ikke ha flere enn 10 siffer' },
-                      })}
-                      label=""
-                      error={errors.values && errors.values[index]?.ytelse?.rentetillegg?.message}
-                      hideLabel={true}
-                    />
-                  </Table.DataCell>
-                </Table.Row>
-              )
-            })}
-          </Table.Body>
-        </Table>
+        <VStack gap="10">
+          <div>
+            <Button
+              variant="secondary"
+              icon={<ArrowsCirclepathIcon />}
+              iconPosition="right"
+              loading={isPending(oppdaterKravgrunnlagStatus)}
+              size="small"
+              onClick={oppdaterKravgrunnlag}
+            >
+              Oppdater perioder fra kravgrunnlag
+            </Button>
+          </div>
+          {mapResult(oppdaterKravgrunnlagStatus, {
+            success: () => <Toast melding="Periodene ble oppdatert" position="bottom-center"></Toast>,
+            error: () => <ApiErrorAlert>En feil oppstod under henting av perioder fra kravgrunnlag</ApiErrorAlert>,
```

Comment:
```suggestion
            error: <ApiErrorAlert>En feil oppstod under henting av perioder fra kravgrunnlag</ApiErrorAlert>,
```
hvis du ikke bryr deg om error-objektet

Reply:
La til detail der jeg. Sikkert greit å ha med.